### PR TITLE
iocage plugin aMule

### PR DESCRIPTION
**Pull request by alberanid**

### CODE REVIEW

**Review by sonicaj — COMMENTED** [review]
Thank you @alberanid for your contribution. Can you please also have a peek at the failing CI test apart from the comments ? Thanks!

File: amule.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "amule",
+  "plugin_schema": "2",
+  "release": "12.2-RELEASE",
```
[the remark this answers is not in the document]
Comment (alberanid) [reply amule.json]: thanks for your remarks, fixed! Regarding the failing CI test, I'll try to look at it in the next days/weeks.

**Comment by alberanid** [comment]
CI test passed setting the SKIP_SERVICE_CHECK=true env.

After that I tried to update the image family for the Cirrus test, but it failed (I presume because I forgot to push the 12.3-RELEASE branch in my repository).

Is it possible to run again the CI test without a new commit?

Thanks

**Comment by sonicaj** [comment]
@alberanid unfortunately it seems like i am not able to. Can you perhaps just change the last commit hash and force push again ? That would trigger it again

**Comment by alberanid** [comment]
> @alberanid unfortunately it seems like i am not able to. Can you perhaps just change the last commit hash and force push again ? That would trigger it again

Good idea: it worked!

**Review by sonicaj — APPROVED** [review]
Awesome, thanks @alberanid!

**Pull request merged**